src/Type/Php/ParseUrlFunctionDynamicReturnTypeExtension.php

```diff
@@ -122,17 +122,17 @@ private function cacheReturnTypes(): void
 		}
 
 		$string = new StringType();
-		$integer = new IntegerType();
+		$port = new IntegerRangeType(0, 65535);
```

Comment:
should use `IntegerRangeType::fromInterval(...)`

Reply:
interesting, brb opening a PR that prevents that :)

Reply:
I would've never guessed that! Force-pushed with the requested change.